src/storage/in_mem_storage_structure/in_mem_column_chunk.cpp

```diff
@@ -8,16 +8,28 @@ using namespace kuzu::common;
 namespace kuzu {
 namespace storage {
 
+PropertyCopyState::PropertyCopyState(const LogicalType& dataType) {
+    if (dataType.getLogicalTypeID() == LogicalTypeID::STRUCT) {
+        auto numFields = StructType::getNumFields(&dataType);
+        childStates.resize(numFields);
+        for (auto i = 0u; i < numFields; i++) {
```

Comment:
We currently don't support struct properties on rel. You can remove this code path.